src/components/web3-provider.js

```diff
@@ -25,6 +25,8 @@ const mainnetDappBaseUrl =
   process.env.MAINNET_DAPP_BASEURL || 'https://dapp.originprotocol.com'
 const rinkebyDappBaseUrl =
   process.env.RINKEBY_DAPP_BASEURL || 'https://demo.staging.originprotocol.com'
+const instructionsUrl =
+  process.env.INSTRUCTIONS_URL || 'https://medium.com/originprotocol/origin-demo-dapp-is-now-live-on-testnet-835ae201c58'
```

Comment:
This blog post mentions "demo" and "Rinkeby". For prod environment, is the thinking to point to a different blog or web page ?

Reply:
Yes, the thought is that we haven't written the blog post about mainnet launch yet (so we don't know the URL), so we should make it configurable. 

That said, perhaps we should just change this default to the origin homepage or something for now, rather have _any_ reference to such an old blog post?

Reply:
+1 to pointing to www.originprotocol.com
And later this week we can maybe whip out a quick INSTRUCTIONS.md hosted on github explaining the steps to install metamask and using the DApp in general ?

Reply:
Cool, I'll change the URL. I suspect we'll have a medium post to explain everything? I know Matt doesn't like non-tech people to ever end up on Github. :)